src/vm/i386/asmhelpers.S

```diff
@@ -472,10 +472,12 @@ NESTED_END CallDescrWorkerInternal, _TEXT
 #ifdef _DEBUG
 // int __fastcall HelperMethodFrameRestoreState(HelperMethodFrame*, struct MachState *)
 LEAF_ENTRY HelperMethodFrameRestoreState, _TEXT
+    #define     RETN 8
```

Comment:
@janvorli It seems that __fastcall in Core CLR is same as __stdcall:
```
$ grep -Rn '__fastcall' | grep '#define'
inc/gcdump.h:24:#define FASTCALL __fastcall
inc/gcinfotypes.h:329:#define FASTCALL __fastcall
pal/inc/pal_mstypes.h:61:#define __fastcall      __stdcall
pal/inc/pal_mstypes.h:63:#define _fastcall       __fastcall
vm/fcall.h:379:#define F_CALL_CONV __fastcall
```

Could you let me known when PAL_STDCPP_COMPAT is defined? It seems that PAL_STDCPP_COMPAT is not defined for VM code.